CppCoreGuidelines.md

```diff
@@ -18382,7 +18382,7 @@ Templates provide a general mechanism for compile-time programming.
 
 Metaprogramming is programming where at least one input or one result is a type.
 Templates offer Turing-complete (modulo memory capacity) duck typing at compile time.
-The syntax and techniques needed are pretty horrendous.
+The syntax and techniques needed are horrible.
```

Comment:
This is not "pretty" as in "beautiful", it is an adverb.

Reply:
reverted